### PR TITLE
Add a loading animation to the newsfeed

### LINKED ISSUE #170: Loading animation for newsfeed
The newsfeed always takes a few seconds to load so it would be good if some sort of loading animation could show while this happens.

### PR DESCRIPTION
**Pull request by BoatsAreRockable**
This fixes #170 using an adapted version of @le717's [animation](http://codepen.io/le717/pen/yYZOZR).

### CODE REVIEW

**Comment by le717** [comment]
Even though you `display: none`, the animation is still playing. Either add `animation-play-state: paused` to the hidden class or (possibly the better idea), just `.remove()` the animation from the DOM when done.

File: app/index.html [hunk 1]
```diff
@@ -18,7 +18,13 @@
                     <h2><i class="fa fa-newspaper-o"></i>Recent News</h2>
                 </div>
                 <ul>
-                    <li><div class="container-news"></div></li>
+                  <li>
+                    <div class="container-news">
+                      <div class="loading-dots">
+                        <h1 class="dot one">.</h1><h1 class="dot two">.</h1><h1 class="dot three">.</h1>
```
Comment (le717) [inline app/index.html]: I don't remember having these as `h1` elements, but I guess I did? `span`s would be more fitting.

Comment (BoatsAreRockable) [inline app/index.html]: @le717 yeah they were `h1`s. My guess is that you might of done it so you didn't have to change the font size in CSS. Will change to `span`s though.

**Comment by BoatsAreRockable** [comment]
@le717 I've just pushed some changes to this.

File: app/js/newsfeed.js [hunk 2]
```diff
@@ -119,7 +119,7 @@ module.exports = {};
    * @returns {Boolean} Always returns true.
    */
   function displayNews(posts) {
-    loadingDots.classList.add("hidden");
+    loadingDots.parentElement.removeChild(loadingDots);
```
Comment (le717) [inline app/js/newsfeed.js]: Did you check if `loadingDots.remove()` works

Comment (BoatsAreRockable) [inline app/js/newsfeed.js]: @le717 sorry I hadn't realised `.remove()` was an actual method! Just committed again using `remove()` which does indeed work. I'd suggest squash merging this particular branch when the time comes.

File: app/js/newsfeed.js [hunk 3]
```diff
@@ -119,7 +119,7 @@ module.exports = {};
    * @returns {Boolean} Always returns true.
    */
   function displayNews(posts) {
-    loadingDots.parentElement.removeChild(loadingDots);
+    loadingDots.remove();
```
Comment (le717) [inline app/js/newsfeed.js]: `.remove()` is a long-needed shortcut to what you had. See also: https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove

**Pull request merged**